app/routes/planets_path.py

```diff
@@ -0,0 +1,34 @@
+#ignore this file 
+# we were not sure if we should create a new folder specifically for path routes, 
+#wanted to ask in class but for the meantime used the routes.py folder for paths
+#in case that was what was expected. 
```

Comment:
Since you moved the `routes.py` into a `routes` folder, we usually do that so that we can group together multiple routes files (usually split by resource type). Accessing `routes.py` in a `routes` package folder can feel rather redundant (`from .routes.routes import planet_bp`) so yes, I would move the routes into a route file named for the main resource type (either `planet_routes.py` or `planets_routes.py`).

Reply:
perfect changing that now